src/app/file-browser/components/file-viewer/file-viewer.component.ts

```diff
@@ -121,6 +122,8 @@ export class FileViewerComponent implements OnInit, OnDestroy {
         AccessRole.Editor
       ) && !route.snapshot.data?.isPublicArchive;
 
+    this.canEditFieldsOnFullscreen = this.canEdit && this.canEditOnFullScreen();
```

Comment:
I'm a bit confused on the purpose of this variable. Is there ever a case where `canEdit` and `canEditFieldsOnFullscreen` are going to be different values? I'm not sure if there's a special case but I would think that anything that fullscreen editing shouldn't have different permissions from normal editing (especially since `canEdit` already checks if the file in question is a Public Archive or not).

Reply:
@meisekimiu  good point, but the ticket was suggesting adding the text only in fullscreen mode, not in the file list viewer, that's my reasoning. That is why I am making use of the routes. Should I change it?

Reply:
Outcome: we'll use `canEdit` everywhere.